src/test/ui/lifetimes/issue-79187-2.nll.stderr

```diff
@@ -0,0 +1,44 @@
+error: lifetime may not live long enough
+  --> $DIR/issue-79187-2.rs:9:24
+   |
+LL |     take_foo(|a: &i32| a);
+   |                  -   - ^ returning this value requires that `'1` must outlive `'2`
+   |                  |   |
+   |                  |   return type of closure is &'2 i32
+   |                  let's call the lifetime of this reference `'1`
```

Comment:
Not sure if we have the information available anywhere, but could emit a diagnostic of this quality for the `the required lifetime does not necessarily outlive the anonymous lifetime #1 defined on the body`, too?